CONTRIBUTING.md

```
@@ -23,7 +23,29 @@ We welcome contributions from the community. Before opening a pull request:
 5. Run `make test-race` to confirm all tests pass.
-6. Open a pull request with a clear description of the change.
+6. Sign off every commit (`git commit -s`) — see [Developer Certificate of Origin](#developer-certificate-of-origin) below.
+7. Open a pull request with a clear description of the change.
+
+## Developer Certificate of Origin
+
+All commits must be signed off, certifying the [Developer Certificate of Origin](https://developercertificate.org/): a statement that you wrote the code, or otherwise have the right to submit it under this project's license.
+
+Sign off by adding the `-s` flag when committing:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)